src/celestia/helper.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _HELPER_H_
+#define _HELPER_H_
+
+class Body;
+class Selection;
+
+class Helper
+{
+    public:
+        static bool hasPrimary(Body* body);
```

Comment:
Please fix indents: before `public` & `private` we use only 1 space, and 4 before methods.

Add `const` to `Body*` to tell a compiler that we don't modify `body`.